styles/product-page.css

```diff
@@ -0,0 +1,355 @@
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+.small-space{
+    margin-bottom: 1rem;
+}
+.nav-sublists ul{
+    display: contents;
+}
+
+.content-area{
+    display: flex;
+    flex-direction: column;
+    margin: 2rem 0 0 2rem;
+}
+.card-component-area{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.basic-card-container{
+    background-color: var(--secondary-color);
+    padding: 1rem;
+    border-radius: 0.3rem;
+    margin: 1rem;
+    max-width: 27rem;
+    box-shadow: 1px 1px 15px 0 var(--tertiary-color);
+}
+.basic-card-container.horizontal{
+    display: flex;
+    background-color: var(--secondary-color);
+    padding: 1rem;
+    border-radius: 0.3rem;
+    margin: 1rem;
+    max-width: 40rem;
+    box-shadow: 1px 1px 15px 0 var(--tertiary-color);
+}
+.card-photo.horizontal{
+    width:100%;
+    height: auto;
+    margin: 2rem;
+    max-width: 18rem;
+    max-height: 20rem;
+    border-radius: 0.3rem;
+}
+.photo-area.horizontal{
+    max-width: 45rem;
+    max-height: 30rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+}
+.card-footer{
+    margin-top: 2rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+.price{
+    display: flex;
+}
+.card-subheading.cross{
+    font-size: 1rem;
+    text-decoration: line-through;
+}
+.photo-area{
+    max-width: 27rem;
+    max-height: 30rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+}
+.card-heading{
+    font-size: 2rem;
+    margin: 0.5rem;
+}
+.card-subheading{
+    font-size: 1.5rem;
+    margin: 0 0 0.5rem 0.5rem;
+}
+.card-body{
+    margin: 0 0.5rem 0.5rem 0.5rem;
+}
+.card-footer.horizontal{
+    margin-top: 2rem;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+}
+.photo-badge.horizontal{
+    width: 1.5rem;
+    height: 1.5rem;
+    background-color: var(--secondary-color);
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items:center ;
+    position: absolute;
+    margin-top: 3rem;
+    margin-right: 3rem;
+}
+.btn{
+    font-size: 1rem;
+    border: none;
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    cursor: pointer;
+}
+.btn.wo-outline{
+    color: var(--primary-color);
+    background-color: var(--secondary-color);
+}
+.card-photo{
+    width:100%;
+    margin-top: 2rem;
+    max-width: 15rem;
+    max-height: 25rem;
+    border-radius: 0.3rem;
+}
+.photo-badge{
+    width: 1.5rem;
+    height: 1.5rem;
+    background-color: var(--secondary-color);
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items:center ;
+    position: absolute;
+    margin-top: 3rem;
+    margin-right: 1.5rem;
+}
+.photo-comp{
+    display:flex;
+    justify-content: flex-end;
+}
+.fa-heart{
+    color: red;
+}
```

Comment:
I think you have already declared this **class** in **index.css** . Re-check it once